neighborhood/rest_client.py

```diff
@@ -0,0 +1,34 @@
+import requests
+
+
+class NeighborhoodApiClient(object):
+    def __init__(self, config) -> None:
+        self.root = config["ROOT"]
+
+        super().__init__()
+
+    def _get_actions(self, resource: str = None, **extra_params):
+        url = self.root
+        jsonParams = {"outputFormat": "json"}
+
+        actions = {
+            "FeatureCollection": {
+                "list": {
+                    "method": "GET",
+                    "url": url,
+                    "params": {"request": "getFeature", **jsonParams, **extra_params},
+                },
+            }
+        }
+
+        return actions[resource]
+
+    def neighborhood_list(self):
+        actions = self._get_actions(
+            "FeatureCollection", **{"typeName": "avoindata:Kaupunginosajako"}
+        )
+        return requests.request(
+            actions["list"]["method"],
+            actions["list"]["url"],
+            params=actions["list"]["params"],
+        )
```

Comment:
How likely it is that there will be another action(s) added that will actually benefit from this `get_actions()` thingy, or is there some other reason for it? Right now this seems just like a very complex way to execute one request 🙂 I would just do the request in one simple method now, and add complexity later when/if needed.

Reply:
Maybe you are right. I did it like this so it would work the same way as the linkedEvents rest_client does. But I don't see it would grow from this, so it could be simplified.

Reply:
Simplified it by removing get_actions and the tests related to it.